Mathlib/Data/Nat/Cast/Order.lean

```diff
@@ -114,6 +141,42 @@ theorem cast_lt_one : (n : α) < 1 ↔ n = 0 := by
 theorem cast_le_one : (n : α) ≤ 1 ↔ n ≤ 1 := by rw [← cast_one, cast_le]
 #align nat.cast_le_one Nat.cast_le_one
 
+variable [m.AtLeastTwo] [n.AtLeastTwo]
+
+-- See note [no_index around OfNat.ofNat]
+@[simp]
+theorem ofNat_le :
```

Comment:
Arguably we need versions with `ofNat` on one side and `Nat.cast` on the other too...

Reply:
Good point. Will add